command/deployment/add.go

```diff
@@ -44,14 +44,12 @@ var CommandAdd = &cli.Command{
 			EnvVars: []string{"VELA_REF", "DEPLOYMENT_REF"},
 			Name:    "ref",
 			Usage:   "provide the reference to deploy - this can be a branch, commit (SHA) or tag",
-			Value:   "refs/heads/master",
 		},
 		&cli.StringFlag{
 			EnvVars: []string{"VELA_TARGET", "DEPLOYMENT_TARGET"},
 			Name:    "target",
 			Aliases: []string{"t"},
 			Usage:   "provide the name for the target deployment environment",
-			Value:   "production",
```

Comment:
Can you help me understand more about this change?

I ask because GitHub's default is `production`:

https://docs.github.com/en/rest/deployments/deployments?apiVersion=2022-11-28#create-a-deployment

> The `environment` parameter allows deployments to be issued to different runtime environments. Teams often have multiple environments for verifying their applications, such as `production`, `staging`, and `qa`. This parameter makes it easier to track which environments have requested deployments. The default environment is `production`.

Reply:
@ecrupper might chime in too, but it was mainly because we thought "production" was a bit too bold. i'm actually surprised it's the default value on GitHub's end.

if we keep this change, we will want to update the UI code as well though.

Reply:
Yeah I agree that we should just default to whatever the SCM's default is. I'm also surprised it's `production`. 

As long as we're interpreting the `target` all the same (what we get in the payload), which it [looks like we are](https://github.com/go-vela/server/blob/9b84d35cb82c08881535d4a4be48b61f7280739e/scm/github/webhook.go#L319), then this wouldn't even be breaking, I guess.